policy_definitions/tagging/policy_assignment_qby_deploy_require_tags.tmpl.json

```
@@ -0,0 +1,20 @@
+{
+    "name": "QBY-Deploy-Require-Tags",
```

As this policy does not deploy anything. I think require tags is enough
```suggestion
    "name": "QBY-Require-Tags",
```

---

Changed as suggested.